conf/drupal/config/webform.webform.session_feedback.yml

```diff
@@ -152,7 +152,8 @@ access:
     users: {  }
     permissions: {  }
   view_own:
-    roles: {  }
+    roles:
+      - authenticated
```

Comment:
@Brian-Clement this is unrelated to the submissions block, right? It's just so users can see what they submitted?

Reply:
Sorry @froboy, hadn't gotten around to writing up some comments here (sorry for jumping in and pushing to your branch like this). 

The crux of what I wanted to do was to leverage `hook_webform_submission_access()` to conditionally grant users permission to view results (based on the logic you introduced in `hook_block_access()`).  The strange thing is that `hook_webform_submission_access()` was never firing at all until I added some some `view` permissions to this particular form.

Once I added this I was able to start seeing `hook_webform_submission_access()` at play so I could start adding logic there.  The disabling of SQL rewrite in the View was also done for this same purpose--sort of a workaround that lead to being able to bypass some of the inner workings of webform's permissions.